src/Tizen.WebView/Tizen.WebView/BackForwardList.cs

```diff
@@ -0,0 +1,170 @@
+            }
+        }
+
+        /// <summary>
+        /// Title of the back forward list item.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string Title
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_back_forward_list_item_title_get(_item_handle);
+            }
+        }
+
+        /// <summary>
+        /// Original Url of the back forward list item.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string OriginalUrl
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_back_forward_list_item_original_url_get(_item_handle);
+            }
+        }
+    }
+
+    /// <summary>
+    /// This class provides the properties of Back Forward list of a specific WebView.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class BackForwardList
+    {
+        private IntPtr _list_handle;
+
+        /// <summary>
+        /// Creates a Back Forward List object.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        internal BackForwardList(IntPtr handle)
+        {
+            _list_handle = handle;
+        }
+
+        /// <summary>
+        /// Gets current item of the back forward list.
+        /// </summary>
+        /// <returns>The current backforward list item.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public BackForwardListItem GetCurrentItem()
+        {
+            IntPtr ItemPtr = Interop.ChromiumEwk.ewk_back_forward_list_current_item_get(_list_handle);
+            BackForwardListItem item = new BackForwardListItem(ItemPtr);
+            return item;
+        }
+
+        /// <summary>
+        /// Gets previous item of the back forward list.
+        /// </summary>
+        /// <returns>The previous backforward list item.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public BackForwardListItem GetPreviousItem()
+        {
+            IntPtr ItemPtr = Interop.ChromiumEwk.ewk_back_forward_list_previous_item_get(_list_handle);
+            BackForwardListItem item = new BackForwardListItem(ItemPtr);
+            return item;
+        }
+
+        /// <summary>
+        /// Gets the back forward list count.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public uint Count
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_back_forward_list_count(_list_handle);
+            }
+        }
+        /// <summary>
+        /// Gets the list containing the items preceding the current item
+        /// limited by limit.
+        /// </summary>
+        /// <param name="limit"> limit The number of items to retrieve.</param>
+        /// <returns>The array list of the back items.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public ArrayList CopyBackItems(int limit)
```

Comment:
```suggestion
        public IList<BackForwardListItem> CopyBackItems(int limit)
```

Reply:
We will accept your suggestion.

Reply:
I have done with IList please check whether it is proper